.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@EugenMayer Could you please remove the `.idea` directory from the PR and add `.idea` to the `.gitignore`?